app/models/state.rb

```diff
@@ -1,5 +1,6 @@
 class State < ApplicationRecord
   has_many :cities
+  has_many :parks
```

Comment:
Please update this unit test

Reply:
Updated `park_spec.rb` to have a state ID